src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -403,3 +411,10 @@ class AttachedAccountRoot implements ISubscriptionContext {
         throw this._error;
     }
 }
+
+async function delay(milliseconds: number): Promise<void> {
```

Comment:
At some point we want to put this in the utils folder because it's in like 3 files in this repo now

Reply:
That's a good idea, I did notice it was mentioned in couple places.